update instructions to add a blueprint (#4371)
camunda/camunda-docs: docs/components/modeler/web-modeler/launch-web-modeler.md

2. Select **Create new project** to create a new project and store diagrams.
   ![web modeler empty home](img/web-modeler-new-user-home.png)
3. Name your diagram. You can go back and change the name any time by clicking on the project name and **Edit name**.

> 4. Select **Browse blueprints** to open the blueprints dialog and view blueprints for various use cases as a starting point for your first diagram. Open these blueprints by selecting **Use blueprint**, which will download the blueprint into the project and open it in the diagram screen. Alternatively, click **Create new > BPMN diagram** to create a blank BPMN diagram.

Reviewer: ```suggestion
4. Select **Browse blueprints** to open the blueprints dialog and browse blueprints for various use cases as a starting point for your first diagram. Open a blueprint by selecting **Use blueprint**, which downloads the blueprint into the project and opens it in the diagram screen. Alternatively, click **Create new > BPMN diagram** to create a blank BPMN diagram.
```
Rewording suggestion? Also, possibly too much in this single step, could use a sub-bulleted list?

Author: I'll apply your changes and move a few things around -- also realized I still need to update the screenshot, so we can keep this on standby for a short while.
